android/README.md

```diff
@@ -0,0 +1,50 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/sdk-sample-apps">
+    <img src="https://cdn.forgerock.com/logo/interim/Logo-PingIdentity-ForgeRock-Hor-FullColor.svg" alt="Logo">
+  </a>
+  <hr/>
+</p>
+
+## Samples
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping Android SDK please look at one of our samples:
+
+- [**Java Authenticator - `/java-authenticator`**](./java-authenticator/)
+  
+  - This Authenticator sample app supports registration of multiple accounts and multiple different authentication methods in each account, such as push notifications and one-time passwords.
+
+- [**Java UI prototype - `/java-ui-prototype`**](./java-ui-prototype/)
+  
+  - An example Android project making full use of the Ping SDK with a prototyping UI. Allows to explore the SDK uses in detail with example calls for running Journeys, getting OAuth2.0 tokens, User Info and more. 
```

Comment:
Let's avoid "Ping SDK" and just go with "SDK" to avoid any renames/clashes.

Reply:
See comment above